src/Apps/Order/Routes/Shipping2/SavedAddresses2.tsx

```diff
@@ -68,6 +61,11 @@ const SavedAddresses: React.FC<SavedAddressesProps> = props => {
     selectedAddressID && getAddressByID(addressList, selectedAddressID)
   const selectedAddressPresent = !!selectedAddress
 
+  const {
+    clickedShippingAddress,
+    clickedAddNewShippingAddress,
+  } = useOrderTracking()
```

Comment:
[blocking]: We should avoid destructuring here (and elsewhere in the PR) 🙏 I mentioned this in @erikdstock's  previous refactor PR, but for complex code that is in the process of a refactoring pass we need all of the helpful clarifying metadata we can get. Something like
```tsx
tracking.clickedShippingAddress()
```
tells us very precisely whats going on, where a bare invocation like 
```
clickedShippingAddress()
```
could mean many things at a glance, and future devs might get confused thinking its some kind of event handler that performs further operations. 

An alternative (that preserves destructuring) would be to rename the functions to 
```tsx
const {
  trackClickedShippingAddress
  trackAddNewShippingAddress
} = useOrderTracking() 
```

Though I would encourage the first approach over this.